apps/tools/src/app/page.tsx

```diff
@@ -0,0 +1,92 @@
+"use client";
+import { FunctionComponent } from "react";
+import { Stack, Typography } from "@mui/material";
+import { WalletIcon } from "@newm-web/assets";
+import theme from "@newm-web/theme";
+
+export interface Tool {
+  readonly Icon: FunctionComponent;
+  readonly description: string;
+  readonly name: string;
+  readonly path: string;
+  readonly title: string;
+}
+
+export const NEWM_TOOLS: Tool[] = [
+  {
+    Icon: WalletIcon,
+    description:
+      "After connecting, you will be able to connect your wallet to the mobile app",
+    name: "wallet-connect",
+    path: "/wallet-connect",
+    title: "Connect your wallet",
+  },
+];
+
+const Page: FunctionComponent = () => {
+  return (
+    <Stack aria-label="NEWM Tools" mt={ 4 } role="group">
+      <Stack
+        component="ul"
+        display="grid"
+        gap={ 3 }
+        gridTemplateColumns={ ["1fr", "1fr 1fr", "1fr 1fr 1fr"] }
+      >
```

Comment:
Maybe this should be centered currently, The one wallet-connect app left aligned seems off